README.md

```diff
@@ -40,3 +42,31 @@ isochrone(location, time, resolution, network, function(err, drivetime) {
 ```
 
 Alternativaly the `network` parameter can be an [OSRM](https://github.com/Project-OSRM/node-osrm) module instance. Allowing setup an OSRM with custom paramters, e.g. usage of shared-memory.
+
+You can too define your own function to draw line/polygon instead of default:
+
+```js
+var concave = require('turf-concave');
+var Isochrone = require('osrm-isochrone');
+
+var resolution = 25; // sample resolution
+var time = 300; // 300 second drivetime (5 minutes)
+var maxspeed = '70'; // in 'unit'/hour
+var unit = 'miles'; // 'miles' or 'kilometers'
+var network = './dc.osrm' // prebuild dc osrm network file
+var location = [-77.02926635742188,38.90011780426885]; // center point
+
+var isochrone = new Isochrone(location, time, resolution, maxspeed, unit, network, function(err, drivetime) {
+  if(err) throw err;
+  // your geojson from draw overload
+  console.log(JSON.stringify(drivetime))
+});
+isochrone.draw = function(destinations) {
```

Comment:
My opinion is it is simpler and clearer like that. It is possible to add a (8th!) function parameter for the exports, but draw function is strongly linked to the data built in isochrone (ie the grid) : it is the reason why I've added bbox and cellGridSize to ```this```.
If it is not the case, you have to send params to draw functions but each draw function maybe needs a different param (and a different signature...).

It's more elegant (and clearer) to use object syntax instead of function syntax ;) 

Fabien